src/components/CGallery/index.meta.js

```diff
@@ -34,10 +34,6 @@ export default {
       key: 'carousel',
       name: 'Carousel',
     },
-    data: {
-      key: 'data',
-      name: 'Source',
-    },
```

Comment:
Can you just explain how did this become unnecessary 🤔 ?
Does it mean that we will no longer have `Data` rendered as a group inside Builder options or...?

Reply:
This file: https://github.com/chmjs/chameleon-vuetify/blob/master/src/index.meta.js has "default" group definition. And in the end (inside builder) every element optionGroups, are merged with this default ones.. So I've actually removed any mention, of this default ones, from components meta

Reply:
Forgot about that, great 👍